scripts/Tools/cimeteststatus

```diff
@@ -317,52 +318,52 @@ def findSpecs(args):
         testspecfiles.append(specfile)
     else:
         testspecfiles = glob.glob("testspec*xml")
-    
+
     for file in testspecfiles:
         if not os.path.isfile(file):
             print("could not find test spec file " + file + ", aborting")
             sys.exit(1)
-        
-    
+
+
     return testspecfiles
-            
+
 
 def getSuiteInfo(specfile):
-    """ Read the test spec, get relevant test info 
+    """ Read the test spec, get relevant test info
     """
 
     tree = etree.parse(specfile)
     root = tree.getroot()
     suiteinfo = {}
     testlist = []
-    
+
     for tr in root.findall('testroot'):
-        suiteinfo['testroot'] = tr.text    
-    
+        suiteinfo['testroot'] = tr.text
+
     for cr in root.findall('cimeroot'):
         suiteinfo['cimeroot'] = cr.text
-    
+
     for bltag in root.findall('baselinetag'):
         suiteinfo['baselinetag'] = bltag.text
-    
+
     for compiler in root.findall("./testlist/compiler"):
         suiteinfo['compiler'] = compiler.text
-    
+
     for mpilib in root.findall('mpilib'):
         suiteinfo['mpilib'] = mpilib.text
-    
+
     for t in root.findall('test'):
-        testlist.append(t.attrib['case'])
+        testlist.append(t.get('case'))
```

Comment:
Yes, the .get one should be preferred in most cases. The .attrib one can be used if you want to throw an exception if the attribute is missing.